index.js

```diff
@@ -94,20 +101,18 @@ class SlackNotifier extends NotificationBase {
             return;
         }
         const pipelineLink = buildData.buildLink.split('/builds')[0];
-
+        const truncatedSha = buildData.event.sha.slice(0, 6);
         // eslint-disable-next-line max-len
-        const message = `<${pipelineLink}|${buildData.pipeline.scmRepo.name}#${buildData.jobName}> *${buildData.status}*`;
+        const message = `*${buildData.status}* ${STATUSES_MAP[buildData.status]} <${pipelineLink}|${buildData.pipeline.scmRepo.name} ${buildData.jobName}>`;
         const attachments = [
             {
                 fallback: '',
                 color: COLOR_MAP[buildData.status],
-                fields: [
-                    {
-                        title: 'Build',
-                        value: `<${buildData.buildLink}|#${buildData.build.id}>`,
-                        short: true
-                    }
-                ]
+                title: `#${buildData.build.id}`,
+                title_link: `${buildData.buildLink}`,
+                // eslint-disable-next-line max-len
+                text: `${buildData.event.commit.message} (<${buildData.event.commit.url}|${truncatedSha}>)` +
```

Comment:
You will have to truncate the commit message. We've seen cases where commit message overflows. 

That was the reason we did this fix for UI https://github.com/screwdriver-cd/ui/pull/399/files#diff-007d51a1584c112070731fd1fcdd9504R63